examples/plot_metrics_with_error.py

```diff
@@ -0,0 +1,163 @@
+# %%
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""
+================================
+Plotting Metrics with Error Bars
+================================
+"""
+# %%
+# Load and preprocess the data set
+# --------------------------------
+# We download the data set using `fetch_adult` function in `fairlearn.datasets`.
```

Comment:
This doesn't seem consistent with what's happening below, since `fetch_openml` is used. If this was copied from elsewhere then it would be awesome if you could submit a PR to fix it in the other place, too. I think we do want to use `fetch_adult` from Fairlearn, but in the past the docs weren't versioned so we had to roll back and use `fetch_openml`. Anyway, that's not needed anymore, so we can showcase how to use `fetch_adult`.